helm/soperator-notifier/templates/_helpers.04_rule.tpl

```diff
@@ -52,5 +53,5 @@ job_state=~"{{ include "son.config.jobStatus.completed" . }}"
 
 {{/* MetricsQL selector for system jobs. */}}
 {{- define "son.rule.jobSelector.system" -}}
-user_name!~"^(nebius|soperatorchecks)$"
+user_name!~"^(nebius|soperatorchecks)$", {{- if .Values.slack.forceUserMail}} user_mail!=""{{ end -}}
```

Comment:
I think it's not the best place to put this filter. Much better place would be in `ignoreFilters` [section](https://github.com/nebius/soperator/blob/main/helm/soperator-notifier/templates/04_rule.yml#L1-L4).
It just seems odd to filter by user email inside system job selector.

Reply:
you're right, thanks